content/en/docs/tasks/administer-cluster/namespaces.md

```diff
@@ -189,8 +189,7 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
     To demonstrate this, let's spin up a simple Deployment and Pods in the `development` namespace.
 
     ```shell
-    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development
-    kubectl scale deployment snowflake --replicas=2 -n=development
+    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development --replicas=2
     ```
     We have just created a deployment whose replica size is 2 that is running the pod called `snowflake` with a basic container that just serves the hostname.
     Note that `kubectl run` creates deployments only on Kubernetes cluster >= v1.2. If you are running older versions, it creates replication controllers instead.
```

Comment:
Also remove this line and the one after it. Both generators and ReplicationController resources are superseded.

Reply:
Done